app/helpers/application_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
tìm hiểu và sử dụng `I18n in rails ` cho text `"Ruby on Rails Tutorial Sample App"`